.devcontainer/devcontainer.json

```
@@ -13,7 +17,7 @@
 
 	// Use 'postCreateCommand' to run commands after the container is created.
 	//https://github.com/microsoft/vscode-dev-containers/issues/559
-	"postCreateCommand": "source $NVM_DIR/nvm.sh && nvm install 20"
+	"postCreateCommand": "npm install && bash"
```

```suggestion
	"postCreateCommand": "npm install"
```

`postCreateCommand` is for setting up the environment, not for launching a shell.